_episodes/01-learner-profile.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Why should I take this course?"
+teaching: 8
+exercises: 7
+questions:
+- "Why would I be taking this course?"
+- "What can I expect from this course?"
+objectives:
+- "Identify performance bottlenecks of running LAMMPS in a HPC cluster"
+- "Be enable to use hardware architecture support at run-time using KOKKOS"
+- "Be able to identify appropriate run-time input parameters to achieve LAMMPS performance acceleration"
```

Comment:
This is a bit of a mouthful, but is this markedly different from the previous point though?

Reply:
Are you commenting on the learner-profile or on the Contents in index.md that I had updated today?
The content has been modified. Well, it is not very different, but definitely more structured. The sequences of the chapters has been decided such that the transition from its previous chapter happens smoothly conceptually.
Do you think the current skeleton is okay? Then, I can quickly start working on it. Anyway, today is my last working day this year. I'll be back on 6th Jan, 2020 again.
Best regards.